### PR TITLE
Ft login with google 187300097

### PR DESCRIPTION
**Pull request by RWEMAREMY**
#### What does this PR do?

login using Google

#### Description of Task to be completed?

-A user should use the Google account to login

#### Any background context you want to provide?

-run "npm run dev" to try it out

#### What are the relevant pivotal tracker/Trello stories?

- id of the project
    #187300097

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
+import { useRouter } from 'next/navigation';
 
 export default function Home() {
+  const router = useRouter();
+  const searchParams = useSearchParams();
+  const token = searchParams.get('token');
+  useEffect(() => {
+    if (token) {
+      localStorage.setItem('token', token);
+      console.log(token);
```
Comment (fab-ryan) [inline src/app/page.tsx]: remove this console

File: src/services/authenticate/login.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      },
+      body: JSON.stringify(formData)
+    });
+
+    const data = await response.json();
+    return data;
+  } catch (err: unknown) {
+    const errors = err as Error;
+    return errors.message;
+  }
```
Comment (fab-ryan) [inline src/services/authenticate/login.ts]: we don't have this file

File: src/services/authenticate/register.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      },
+      body: JSON.stringify(formData)
+    });
+
+    const data = await response.json();
+    return data;
+  } catch (err: unknown) {
+    const errors = err as Error;
+    return errors.message;
+  }
```
Comment (fab-ryan) [inline src/services/authenticate/register.ts]: the same as this meaning you didn't rebase

**Review by fab-ryan — CHANGES_REQUESTED** [review]
rebase to the current branch develop

File: src/services/authenticate/login.ts
Comment (fab-ryan) [inline src/services/authenticate/login.ts:1]: remove this file please

File: src/services/authenticate/register.ts
Comment (fab-ryan) [inline src/services/authenticate/register.ts:1]: same as this

**Review by fab-ryan — CHANGES_REQUESTED** [review]
remove those files

**Review by fab-ryan — APPROVED** [review]

**Review by Sophie20202 — APPROVED** [review]

**Review by musabehonore — APPROVED** [review]

**Review by Ruthuwamahoro — APPROVED** [review]

**Pull request merged**